tests/pos/i4563.scala

```diff
@@ -0,0 +1,12 @@
+case class Foo(str: String)
```

Comment:
This test already compiles and runs on master. TBH, I am not sure this minimises the original issue.

Anyway, this was reported as a runtime issue, so this should be a run test